supported.d.ts

```diff
@@ -0,0 +1,13 @@
+import type { JSONSchema7 } from 'json-schema';
+
+declare module '@asyncapi/specs/supported' {
+  declare const schemas: {
```

Comment:
Could we somehow avoid having to add every new supported version into this object?
Could we do the opposite, to maintain a list of non-supported (or regex, or whatever) and from that build this?

Reply:
I'm not sure I get it. You know, still new into this world 😆 . What is the feature we lose if we do it as I suggested?

Reply:
I tested your idea and TS did not see in other packages the filtered versions without the unsupported ones, instead of `2.0.0` etc it showed me string type and not union. Maybe with TS -> it will see it.

Reply:
No longer current. PR has changed its "character".